VERSION

```diff
@@ -1 +1 @@
-1.10.0-DEV
+1.10.0-alpha1
```

Comment:
Don't think you meant to change this?
```suggestion
1.10.0-DEV
```

Reply:
Thank you. I resolved the conflict by merging in the `master` branch.